lib/books/book.rb

```diff
@@ -0,0 +1,9 @@
+require_relative '../item'
+
+class Book < Item
+  def initialize(publisher, cover_state)
+    super()
```

Comment:
- [ ] Kindly inform on super all the data needed from the parent class. Super is used to inform which data will be used from the parent class.